examples/bucket_status.rs

```diff
@@ -0,0 +1,90 @@
+// Copyright 2025 ReductStore
```

Comment:
There is too little functionality. Don't create examples or update the README file for new features unless it has been requested or mentioned in an issue. Update the 'Agents.md' file.